.circleci/config.yml

```diff
@@ -128,6 +128,17 @@ jobs:
     steps:
       - checkout
       - run: deployment/molecule/molecule.sh ui
+  ultimate-native-to-erc:
+    machine:
+      image: circleci/classic:latest
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - run: git submodule update --init
+      - run:
+          name: Prepare the infrastructure
+          command: e2e-commons/up.sh deploy native-to-erc
```

Comment:
could you clarify what exactly will happen with this step? Will it setup parity nodes and deploy contracts?

Reply:
The `e2e-commons/up.sh deploy native-to-erc` command performs the following:

1. Spins up the parity nodes (`docker-compose up -d parity1 parity2 e2e`)
2. Deploys the contracts (`docker-compose run e2e e2e-commons/scripts/deploy.sh`)
3. Launches the Molecule scenario (`../deployment/molecule/molecule.sh ultimate-native-to-erc`)
    - The Molecule scenario spins up ubuntu container
    - Then executes the oracle deployment playbook on that container
4. (todo) Run the oracle-e2e tests

Let me know if anything needs more clarification. I'm also thinking about adding some kind of diagram to our readmes that would illustrate this, specially the parts with docker-in-docker.

Reply:
Thanks for clarification! Yes, some document describing the molecule testing is definitely needed. Too many relations.

Reply:
When I reviewed the code I lost top and bottom of `up.sh` that's why I did not get where parity is started and arguments are shifted.